### PR TITLE
fix: check page improvement

### PR DESCRIPTION
**Pull request by Saikilito**
### Description

Improvement styles into check page

- #117 
- #118 

#### Screenshots

Adobe XD mobile design 

image

Current mobil dev branch

image

In this mr

image

**Desktop version is update too**

#### Type of change

Please delete options that are not relevant.

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/hooks/useChats.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { logError } from '../helpers'
 export const useLatestMessages = ({ packageId }) => {
   const [LatestMessages, setLatestMessages] = useState([])
   const { data, loading } = useSubscription(GET_CHAT_MESSAGES_SUBSCRIPTION, {
-    variables: { package_id: packageId },
+    variables: { 'package_id': packageId },
```
Comment (heliomar-pena) [inline src/hooks/useChats.js]: Why the single quotes are required? Or is a lint fix?

File: src/hooks/usePackage.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ export const useDasherLatestCoordinates = packageId => {
 export const useGetPackagesIdByCode = packageCode => {
   const [packageInformation, setPackageInformation] = useState({ packages: [] })
   const { loading, error, data } = useQuery(GET_PACKAGE_ID_QUERY_BY_CODE, {
-    variables: { package_code: packageCode },
+    variables: { 'package_code': packageCode },
```
Comment (heliomar-pena) [inline src/hooks/usePackage.js]: Why the single quotes are required? Or is a lint fix?

File: src/hooks/usePackage.js [hunk 3]
```diff
@@ -52,9 +52,9 @@ export const useConfirmPackage = () => {
   const { loading, error, data } = useQuery(CONFIRM_PACKAGE_QUERY, {
     variables: {
       id: packageId,
-      client_name: confirmData.name,
+      'client_name': confirmData.name,
       rut: confirmData.RUT,
-      client_phone: confirmData.phone,
+      'client_phone': confirmData.phone,
```
Comment (heliomar-pena) [inline src/hooks/usePackage.js]: Some keys doesn't have single quotes and others yes. Exists a reason? Please try to maintain a standard

File: src/pages/Confirm/Confirm.jsx [hunk 4]
```diff
@@ -30,7 +30,7 @@ const Confirm = () => {
     event.preventDefault()
     // Submit rating here
     if (rating) {
-      insertClientRate({ variables: { id, client_rating: rating } })
+      insertClientRate({ variables: { id, 'client_rating': rating } })
```
Comment (heliomar-pena) [inline src/pages/Confirm/Confirm.jsx]: Here too are single quotes

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
Some questions

File: src/hooks/useChats.js (hunk 1 above)
Comment (Israel-Laguan) [reply src/hooks/useChats.js]: I addressed that issue

File: src/hooks/usePackage.js (hunk 3 above)
Comment (Israel-Laguan) [reply src/hooks/usePackage.js]: I addressed that issue

File: src/pages/Confirm/Confirm.jsx (hunk 4 above)
Comment (Israel-Laguan) [reply src/pages/Confirm/Confirm.jsx]: I addressed that issue

**Review by heliomar-pena — APPROVED** [review]

File: src/hooks/usePackage.js (hunk 2 above)
Comment (Israel-Laguan) [reply src/hooks/usePackage.js]: I addressed that issue

**Review by Israel-Laguan — APPROVED** [review]

**Pull request merged**